docker/cortx-deploy/generate-release-info.sh

```diff
@@ -30,7 +30,7 @@ VERSION: "$VERSION"
 OS: $(cat /etc/redhat-release | sed -e 's/ $//g' -e 's/^/\"/g' -e 's/$/\"/g')
 DATETIME: $(date +"%d-%b-%Y %H:%M %Z" | sed -e 's/^/\"/g' -e 's/$/\"/g')
 COMPONENTS:
-$(for component in $(sed 's/#.*//g' /opt/seagate/cortx/cortx-componenet-rpms.txt); do echo "    - \"$(rpm -q $component).rpm\"" ; done)
+$(for component in $(sed 's/#.*//g' /opt/seagate/cortx/cortx-componenet-rpms.txt ceph-componenet-rpms.txt); do echo "    - \"$(rpm -q $component).rpm\"" ; done)
```

Comment:
do we need to give full path for `ceph-componenet-rpms.txt`?

Reply:
Fixed